tomviz/Operator.h

```diff
@@ -41,6 +41,13 @@ enum class OperatorState
   ERROR
 };
 
+enum class TransformResult
+{
+  COMPLETE = static_cast<int>(OperatorState::COMPLETE),
```

Comment:
Would it be better to just specify the type of the enum, such as char, and cast to that, or go through the extra hoops to get at the underlying type the compiler automatically selects? This is described in item 10 of Scott Meyer's book.